cognite/client/_api/files.py

```diff
@@ -700,20 +700,30 @@ def upload_bytes(
         return FileMetadata._load(returned_file_metadata)
 
     def retrieve_download_urls(
-        self, id: Union[int, Sequence[int]] = None, external_id: Union[str, Sequence[str]] = None
+        self,
+        id: Union[int, Sequence[int]] = None,
+        external_id: Union[str, Sequence[str]] = None,
+        extended_expiration: bool = False,
     ) -> Dict[Union[int, str], str]:
         """Get download links by id or external id
 
         Args:
-            id (Union[int, Sequence[int]]: Id or list of ids.
-            external_id (Union[str, Sequence[str]]: External id or list of external ids.
+            id (Union[int, Sequence[int]]): Id or list of ids.
+            external_id (Union[str, Sequence[str]]): External id or list of external ids.
+            extended_expiration (optional, bool): Extend exipration time of download url to 1 hour.
```

Comment:
is it going to be documented in service-contracts repo? Don't see it currently https://docs.cognite.com/api/v1/#tag/Files/operation/downloadLinks

Reply:
Yes, it should be. I'll make a PR there aswell.